components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+&= 
+\frac{D_\perp \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{D t} \\
+&=
+\tilde{h}\frac{D_\perp  \overline{  {\bf u} }^{\tilde z} }{D t} 
++
+\overline{ {\bf u} }^{\tilde z} \frac{D_\perp \tilde{h}}{D t} \\
+&= 
+\tilde{h}\left(
+  \frac{\partial  \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left( \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right)  \right)
+- \overline{ {\bf u} }^{\tilde z} \left( 
+   \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_k^{\text{top}}} 
+ - \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_{k+1}^{\text{bot}}}
+\right) 
+$$ (2D-material-der-product)
+
+We now substitute [](#2D-material-der-product) into [](#momentum-layered-differential-1) and divide by $\tilde h$ to get
+
+$$
+ \frac{\partial \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left(  \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+& + \frac{\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} }{\tilde h}
+-   \frac{\overline{ {\bf u} }^{\tilde z}}{\tilde{h}}  \left( 
+   \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_k^{\text{top}}} 
+ - \left[ {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_{k+1}^{\text{bot}}}\right)
+\\ &  =
+-  \overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} 
+-  \overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \frac{1}{\rho_0 \, \tilde{h}} \overline{\nabla_{\perp} p}^{z}  
++  \frac{1}{\rho_0 \, \tilde{h}} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z 
+$$ (momentum-layered-differential-2)
+
+Note that the coefficient of the pressure gradient and stress tensor can be rewritten as
+
+$$
+\frac{1}{\rho_0 \, \tilde{h}} = 
+\frac{1}{\rho_0 \,\frac{1}{\rho_0} \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz}
+= \frac{1} {\overline{\rho}^z}
+= \overline{\alpha}^z,
+$$ (grad-p-coeff)
+
+where $\alpha$ is the specific volume. 
+Gathering the vertical advection terms, [](#momentum-layered-differential-2) becomes
+
+$$
+ \frac{\partial \overline{  {\bf u} }^{\tilde z} }{\partial t} 
+ + \nabla_\perp \cdot \left(  \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+& 
++ \left( \nabla_\perp \cdot  {\bf u}  \right) {\bf u}
++ \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{top}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - 
+  \frac{1}{\tilde h}\left(
+ \left[ {\bf u}\right]_{z=z^{\text{bot}}}-\overline{ {\bf u} }^{\tilde z}\right) \, \left[\tilde{w}_{tr} \right]_{z=z^{\text{bot}}} 
+\\ &  =
+-  \overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} 
+-  \overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \overline{\alpha}^z \overline{\nabla_{\perp} p}^{z}  
```

Comment:
We do not want to make simplifying assumptions about the pressure gradient at this time and instead need to keep the full notation as it was:
```suggestion
-  \overline{\alpha \nabla_{\perp} p}^{\tilde{z}}
```